netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -483,17 +508,20 @@ public void operationComplete(ChannelFuture future) throws Exception {
     });
   }
 
-  private void gracefulClose(ChannelHandlerContext ctx, GracefulCloseCommand msg,
+  private void gracefulClose(ChannelHandlerContext ctx, GracefulCloseCmd cmd,
```

Comment:
The class name or the var?   The class name is shorter because each method on it is duplicated (and easy to mix up)